pkg/driver/controller.go

```diff
@@ -135,6 +135,11 @@ createVolumeResponse:
 			VolumeId:      volName,
 			CapacityBytes: size,
 			VolumeContext: VolumeContext,
+			AccessibleTopology: []*csi.Topology{
+				{
+					Segments: map[string]string{TopologyNodeKey: nodeID},
```

Comment:
why this change? Controller does not have any nodeID, it can run anywhere? And why you want to pin PV to a node?

Reply:
ahh added by mistake, forget that it will set the nodeaffinity in PV .. removed now